php/Terminus/Collections/Backups.php

```diff
@@ -221,41 +206,61 @@ function ($backup) {
         return $backups;
     }
 
-  /**
-   * Sets an environment's regular backup schedule
-   *
-   * @param int $day_number A numerical of a day of the week
-   * @return bool True if operation was successful
-   */
-    public function setBackupSchedule($day_number)
+    /**
+     * Valid backup types
+     *
+     * @return string[] An array of valid elements
+     */
+    public function getValidElements()
+    {
+        return ['code', 'files', 'database', 'db',];
+    }
+
+    /**
+     * Sets an environment's regular backup schedule
+     *
+     * @param array $options Elements as follow:
+     *    string  day  A day of the week
+     *    integer hour Hour of the day to run the backups at, 1 = 01:00 24 = 00:00
+     * @return Workflow
+     */
+    public function setBackupSchedule($options)
     {
-        $daily_ttl   = 691200;
-        $weekly_ttl  = 2764800;
-        $backup_hour = rand(1, 24);
-        $schedule    = [];
+        $daily_ttl = 691200;
+        $weekly_ttl = 2764800;
+        $backup_hour = (isset($options['hour']) && !is_null($options['hour'])) ? $options['hour'] : rand(1, 24);
```

Comment:
We shouldn't set a random here, but just leave it null. This was recently changed in the API: https://github.com/pantheon-systems/titan-mt/pull/15200